code/modules/exploration_crew/research_locator.dm

```diff
@@ -0,0 +1,23 @@
+
+/obj/item/research_disk_pinpointer
+	name = "research disk locator"
+	desc = "A small handheld device that detects nearby research disks. Despite its extremely high sensitivity, the returned signal from research disks is so weak that it only has a short range."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "researchlocator"
+	var/next_use_time = 0
```

Comment:
Why not use our fancy cooldowns here?

Reply:
Eh they do the same thing and I find this much easier to read than the defines